rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search/560_ignored_source.yml

```diff
@@ -0,0 +1,143 @@
+---
+setup:
+  - skip:
+      features: headers
+  - do:
+      indices.create:
+        index: test
+        body:
+          mappings:
+            _source:
+              mode: synthetic
+            properties:
+              object:
+                enabled: false
+
+  - do:
+      headers:
+        Content-Type: application/yaml
```

Comment:
I had to include this because it looks like sometimes the source in in json and sometimes is in yaml. This changes the base64 encoded value I check later in `match`.